l3build-check.lua

```diff
@@ -582,6 +583,7 @@ function runcheck(name, hide)
     if errorlevel == 0 then
       return errorlevel
     end
+    failedengines[#failedengines + 1] = engine
```

Comment:
At the top of the file:
local push = table.insert

push(failed_engines, engine)

Reply:
Why? `push` is not a ommon name for this in Lua, so if at all I would call it `insert`. (Which already exists)

But since the only difference between them is that `table.insert` is slower and IMO less clear, I don't see the point in changing it.

Reply:
Do you mean that
`failedengines[#failedengines + 1] = engine`
is more clear that
`push(failed_engines, engine)`
or
`append(failed_engines, engine)`?

push is a common name for that operation.
Concerning the time loss, it is less than epsilon with respect to the whole process.

Reply:
Yes I think that it's more clear since it exactly describes the operation and is a common idiom in Lua. Especially since using `push` or `append`, while common names for similar operations in general, are rather uncommon in Lua code since the operation for tables is often either written like this or using `table.insert`. In general I strongly dislike creating copies of standard functions with different names since this makes them harder to identify and therefore makes people wonder why the standard function hasn't been used.